haskell/monocle.cabal

```diff
@@ -128,6 +128,7 @@ library
                      , http-types                 >=0.12
                      , json-syntax                >= 0.2
                      , lens
+                     , lens-aeson
```

Comment:
I wanted to show how we can handle raw Value efficiently with these lenses, and I think it's good to know how that works. But I agree, it would be better to create a ConfigDocument data type with a FromJSON instance.